gtests/net/mptcp/mp_reset/mp_reset_multi.pkt

```diff
@@ -31,10 +31,10 @@
 +0      >                                .  1:1(0)           ack 1002                                                 <dss dack8=2002                    dll=0    nocs>
 
 // send rst option.  RST is not set, so it should have no effect.
-+0      <                                .  1002:1002(0)               win 256    <mp_reset 1>
++0      <                                .  1002:1002(0)     ack 1     win 256    <mp_reset 1>
 
 // again, this time with RST but out of window.
-+0.1    <                               R.  3001:3001(0)               win 450    <mp_reset 1>
++0.1    <                               R.  3001:3001(0)     ack 1     win 450    <mp_reset 1>
```

Comment:
Is it possible that when a RST is received, the ACK value is not checked? (Do we not go through `tcp_validate_incoming()` without going through `tcp_ack()` before?) I don't think the ACK value needs to be checked, no? [RFC9293](https://www.ietf.org/rfc/rfc9293.html#name-reset-processing)

So are you sure this modification is needed? I agree it is better to set the correct ACK value but it might not cause the issue you mentioned, or does it?


Note that there are other tests where we send a RST with the wrong ACK value:

- `receive_fastclose_with_rst_multi.pkt`
- `receive_fastclose_with_rst_v4.pkt`

Found with:

    git grep "R\. " -- gtests/net/mptcp | grep -vw ack